src/charm.py

```diff
@@ -18,6 +18,9 @@
     IngressCharmEvents,
     IngressProvides,
 )
+from charms.nginx_ingress_integrator.v0.nginx_route import (  # type: ignore[import]
```

Comment:
Should there be a comment here explaining the `type: ignore`?

Reply:
I just removed all `# type: ignore[import]`, it will not be necessary after I add the mypy configuration to `pyproject.toml`. Thanks!